crates/wasmcloud-host/src/host.rs

```diff
@@ -445,14 +470,14 @@ impl Host {
 
     /// Retrieves the list of capability providers running in the host. This function does
     /// _not_ include any capability providers that may be remotely running in a connected
-    /// lattice
+    /// lattice.
     pub async fn get_providers(&self) -> Result<Vec<String>> {
         self.ensure_started()?;
         let b = MessageBus::from_hostlocal_registry(&self.id);
         Ok(b.send(QueryProviders {}).await?.results)
     }
 
-    /// Perform a raw waPC-style invocation on the given actor by supplying an operation
+    /// Perform a raw [waPC](https://wapc.io)-style invocation on the given actor by supplying an operation
```

Comment:
I'm a little concerned with the `https://wapc.io` link here because the docs on that site are all placeholders, I was going to suggest https://crates.io/crates/wapc instead but I wouldn't want to give off the impression we only support Rust. Thoughts?

Reply:
I like the idea of pointing to the crate. They'll make their way to the wapc documentation one way or another, but at least we control the URLs in the crate and the wapc URL (or documentation) could change without us knowing.